pkgs/development/compilers/ocaml/generic.nix

```diff
@@ -95,7 +95,7 @@ stdenv.mkDerivation (args // {
   preConfigure = optionalString (lib.versionOlder version "4.04") ''
     CAT=$(type -tp cat)
     sed -e "s@/bin/cat@$CAT@" -i config/auto-aux/sharpbang
-  '' + optionalString (stdenv.isDarwin && lib.versionOlder version "4.13") ''
```

Comment:
I think a proper fix is find where AS is defined and update it at the root. My grep-ing skill didn't helped me to find where it was set. I tried `'AS='` (with many false positive) `AS =` `t AS` (for `export AS` but I should have cached something before)